docs/data-sources/dbaas_redisdb_replicaset.md

```diff
@@ -0,0 +1,59 @@
+---
+subcategory: "Database as a Service - RedisDB"
```

Comment:
@avirtopeanu-ionos please change the name of the file to `dbaas_redisdb_replica_set.md`. I know this is annoying but we use `replicaset` only for the resource tag. Internally and in the file structure, we use either `ReplicaSet` or `replica_set` since it improves the clarity.